.github/workflows/monorepo-docker.yml

```diff
@@ -109,3 +111,26 @@ jobs:
           build-args: |
             REGISTRY_COMMIT=${{ env.REGISTRY_VERSION }}
           platforms: ${{ steps.determine-platforms.outputs.platforms }}
+
+      - name: Comment image tags on PR
+        if: github.event_name == 'pull_request' && success()
+        uses: marocchino/sticky-pull-request-comment@v2
+        with:
+          header: monorepo-docker-image
+          message: |
+            ## 🐳 Monorepo Docker Image Built Successfully
+
+            **Image Tags:**
+            ```
+            ${{ steps.meta.outputs.tags }}
+            ```
+
+            **Digest:** `${{ steps.build.outputs.digest }}`
+            ```
+
+      - name: Delete PR comment on failure
+        if: github.event_name == 'pull_request' && failure()
+        uses: marocchino/sticky-pull-request-comment@v2
+        with:
+          header: monorepo-docker-image
+          delete: true
```

Comment:
imo can use the gh cli directly like we do in registry for the check-warp-deploy comment

https://github.com/hyperlane-xyz/hyperlane-registry/blob/2b4aab619318495cf499de0bc391135fb3c606fc/scripts/check-warp-deploy.sh#L74-L85